FE/components/webrtc/SidebarChat.tsx

```diff
@@ -0,0 +1,98 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface SidebarChatProps {
+  isShow: boolean;
+  children: ReactElement;
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  overflow: hidden;
+`;
+
+const RightSidebarSpace = styled.div`
+  transition: width 0.3s;
+  width: 0;
+
+  &.open {
+    width: 300px;
+  }
+
+  .sidebar {
+    transition: transform 0.3s;
+    width: 300px;
+    height: 500px;
+  }
+`;
+
+const SidebarContent = styled.div`
+  position: fixed;
+  bottom: 120px;
+  right: 30px;
+  width: 280px;
+
+  display: flex;
+  align-items: flex-end;
+
+  .item {
+    overflow: auto;
+    max-height: 65vh;
+  }
+`;
```

Comment:
```suggestion
const Wrapper = styled.div`
  display: flex;
  overflow: hidden;
  .right-sidebar-space {
    transition: width 0.3s;
    width: 0;

    &.open {
      width: 300px;
    }

    .sidebar {
      transition: transform 0.3s;
      width: 300px;
      height: 500px;
    }
  }
  .sidebar-content {
  	position: fixed;
    bottom: 120px;
    right: 30px;
    width: 280px;

    display: flex;
    align-items: flex-end;

    .item {
      overflow: auto;
      max-height: 65vh;
    }
  }
`;
```

요렇게 className으로 접근하게 하면 여러개의 styled를 만들 필요가 없습니다~!
또한 styled에 props를 전달하는 경우 Wrapper에만 준 뒤 여러곳에서 쓸 수 있어서 편하게 사용 가능합니다.
사실 취향 차이이긴 해서 원하시는대로 사용하시면 됩니다.

Reply:
아아아 감사합니다 또 하나 배웠네요 수고하셨습니다.
제 코드 상당히 이해하기 힘들었을 수 있는데, 금방금방 하시네요👍